html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div class="text">Email</div>
+      <div>
+        <label for="email"></label>
+        <input type="text" name="email" size="12" />
+      </div>
+      <div>Why is coding cool?</div>
+      <div>
+        <select name="Select an option">
+          <option value="cool">Because it's cool</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <div>
+        <input type="radio" name="language" value="html" />HTML
+        <input type="radio" name="language" value="css" />CSS
+        <input type="radio" name="language" value="javascript" />JavaScript
+      </div>
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" name="sign up" value="yes"/>Sign me up!
+      <div class="text">Comments</div>
```

Comment:
you want a label here as well

Reply:
<img width="694" alt="Screen Shot 2022-05-10 at 12 36 38 PM" src="https://user-images.githubusercontent.com/60913524/167708346-34fd06e6-09a6-4f3e-9560-7733914cba42.png">